vite.config.ts

```diff
@@ -4,15 +4,8 @@ import legacy from '@vitejs/plugin-legacy';
 import react from '@vitejs/plugin-react';
 import { resolve } from 'path';
 import { defineConfig } from 'vite';
-import type { UserConfig as VitestUserConfig } from 'vitest/config';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
-declare module 'vite' {
```

Comment:
This module is not needed. Only old versions of Vite needed. It can be safely removed.